app/src/main/res/values/strings.xml

```diff
@@ -572,4 +572,17 @@
   <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
   <string name="select_a_topic_to_start">Select a Topic to Start</string>
   <string name="revision_navigation_cards_header">Continue Studying</string>
+  <string name="info_tab_content_description">Info tab</string>
+  <string name="revision_tab_content_description">After completing a lesson, revise your concepts in the revision tab.</string>
+  <string name="practice_tab_content_description">Practice tab</string>
+  <string name="lessons_tab_content_description">Start learning here in the lessons tab.</string>
+  <string name="topic_lessons_tab_spotlight_hint">Find all your lessons here</string>
+  <string name="topic_revision_tab_spotlight_hint">Revise your lessons here</string>
+  <string name="first_chapter_spotlight_hint">Tap to start a chapter</string>
+  <string name="exploration_exit_button_spotlight_hint">Exit anytime using this button. We will save your progress.</string>
+  <string name="voiceover_icon_spotlight_hint">Would you like %s to read for you? Tap on this button to try!</string>
+  <string name="voiceover_language_icon_spotlight_hint">Tap to change voiceover language</string>
+  <string name="onboarding_next_button_spotlight_hint">Next</string>
```

Comment:
This needs more context than just "Next". "Next" isn't at all instructional.

Also, I'm a bit worried about the "Continue" button being noticeable on the overlay. Perhaps we should also mention that here since this is the first spotlight that the user will encounter?

Reply:
did you mean "Done" instead of "Continue"?   Proposed new hint: "Use this button to view the next slide. To continue, click on "Done". "

Reply:
Although, in a discussion with @mschanteltc , she suggested not having this spotlight at all.